nuxeo-core/nuxeo-core-test/src/test/resources/OSGI-INF/test-coldstorage-contrib.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.ecm.core.coldstorage.test">
```

Comment:
Seing that there is nothing else than the dummy thumbnail factory contribution here, I would rename the file to `test-dummy-thumbnail-factory-contrib.xml` and the component name as well to `org.nuxeo.ecm.core.dummy.thumbnail.factory.test`.

Then, the contribution could be used elsewhere, independently from cold storage testing.

Reply:
yes, initially this was done because this contribution contained also the `InMemory` blob dispatcher. i will rename it.